store/Cargo.toml

```diff
@@ -17,22 +17,25 @@ doctest = false
 
 [dependencies]
 anyhow = { version = "1.0" }
-clap = { version = "4.3" , features = ["derive"] }
-deadpool-sqlite = { version = "0.6", features = ["rt_tokio_1"]}
+clap = { version = "4.3", features = ["derive"] }
+deadpool-sqlite = { version = "0.6", features = ["rt_tokio_1"] }
 directories = { version = "5.0" }
 figment = { version = "0.10", features = ["toml", "env"] }
 miden-crypto = { workspace = true }
+miden-lib = { workspace = true }
 miden-node-proto = { path = "../proto" }
 miden-node-utils = { path = "../utils" }
 miden_objects = { workspace = true }
 once_cell = { version = "1.18.0" }
 prost = { version = "0.12" }
 rusqlite = { version = "0.29", features = ["array", "buildtime_bindgen"] }
 rusqlite_migration = { version = "1.0" }
-serde = { version = "1.0" , features = ["derive"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { version = "1.0" }
```

Comment:
Do we still need `serde` and `serde_json` here?

Reply:
Removed `serde_json`, but we need `serde` because our `Config` system [relies on it](https://github.com/0xPolygonMiden/miden-node/blob/86e3469d192858483ee4be9870f9e1028fd32b59/utils/src/config.rs#L25).